src/DistroHelperUpdates.php

```diff
@@ -178,20 +190,26 @@ public function exportConfig($config_name) {
    *   FALSE if the update failed, otherwise the updated configuration object.
    */
   public function updateConfig(string $configName, array $elementKeys, string $module, string $directory = 'install') {
-    $install_profile_config = DistroHelperUpdates::loadConfigFromModule($configName, $module, $directory)['value'];
+    $new_config = DistroHelperUpdates::loadConfigFromModule($configName, $module, $directory)['value'];
 
-    $config = $this->configManager->getConfigFactory()->getEditable($configName);
-    if ($config->isNew()) {
+    $active_config = $this->configManager->getConfigFactory()->getEditable($configName);
+    if ($active_config->isNew()) {
       // Can't update nonexistent config.
+      $this->logger->error(
+        'No active config found for @configName. Use installConfig to import config that does not already exist in your database.',
```

Comment:
```suggestion
        'No active config found for @configName in updateConfig(). Use installConfig() to import config that does not already exist in your database.',
```
Thinking of seeing this error and needing a little extra context to save my brain cycles.